boot/bootutil/include/bootutil/crypto/ecdsa_p256.h

```diff
@@ -38,6 +38,27 @@
 extern "C" {
 #endif
 
+#if	defined(MCUBOOT_USE_TINYCRYPT) || \
+	defined(MCUBOOT_USE_CC310) || \
+	defined(MCUBOOT_USE_NRF_EXTERNAL_CRYPTO)
+static inline int check_compression_byte(uint8_t **pk)
```

Comment:
The name is a bit deceiving given that it is not just doing a check, but also changing the input.

Reply:
check_and_skip.... get_uncompresed....